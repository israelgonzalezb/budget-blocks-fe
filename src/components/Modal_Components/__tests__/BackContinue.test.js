
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import BackContinue from '../BackContinue';


describe('BackContinue test', () => {
  it('BackContinue should match snapshot', () => {
    const component = renderer.create(<BackContinue
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // hey there...
  });
});
