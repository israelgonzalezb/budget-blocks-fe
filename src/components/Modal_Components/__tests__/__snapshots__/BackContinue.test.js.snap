// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackContinue test BackContinue should match snapshot 1`] = `
<div
  className="MuiDialogActions-root WithStyles(ForwardRef(DialogActions))-root-1 buttons MuiDialogActions-spacing"
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined backBtn MuiButton-outlinedPrimary"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      BACK
    </span>
  </button>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined contBtn MuiButton-outlinedPrimary"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      CONTINUE
    </span>
  </button>
</div>
`;
